litiengine/src/test/java/de/gurkenlabs/litiengine/util/ColorHelperTests.java

```diff
@@ -40,44 +41,34 @@ private static Stream<Arguments> getColorFromHexString() {
         Arguments.of("#0000ff", Color.BLUE));
   }
 
-  @Test
-  void testRedFromAlphaHexString() {
-    String red200 = "#c8ff0000";
-
-    Color redDecoded = ColorHelper.decode(red200);
-
-    Color alphaRed = new Color(255, 0, 0, 200);
-    assertEquals(alphaRed.getRed(), redDecoded.getRed());
-  }
-
-  @Test
-  void testGreenFromAlphaHexString() {
-    String red200 = "#c8ff0000";
+  @Nested
+  class DecodeTest
+  {
+      String red200 = "#c8ff0000";
 
-    Color redDecoded = ColorHelper.decode(red200);
+      Color redDecoded = ColorUtil.decode(red200);
 
-    Color alphaRed = new Color(255, 0, 0, 200);
-    assertEquals(alphaRed.getGreen(), redDecoded.getGreen());
-  }
-
-  @Test
-  void testBlueFromAlphaHexString() {
-    String red200 = "#c8ff0000";
+      Color alphaRed = new Color(255, 0, 0, 200);
```

Comment:
The three above variables should be renamed, as they are no longer only associated with `red`.

I recommend:

`color`
`colorDecoded`
`colorLiteral`

or similar.

Reply:
This suggestion makes more sense if this review is read in split mode:

https://github.com/gurkenlabs/litiengine/pull/819/files/bcd404b55b5335d7cdb1837dd00efb7c8c55038d?diff=split&w=0